corehq/apps/app_manager/static/app_manager/js/manage_releases_by_app_profile.js

```diff
@@ -0,0 +1,96 @@
+hqDefine('app_manager/js/manage_releases_by_app_profile', [
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+    'hqwebapp/js/assert_properties',
+    'translations/js/app_translations',
+], function (
+    $,
+    ko,
+    _,
+    initialPageData,
+    assertProperties
+) {
+    'use strict';
+    $(function () {
+        var AppRelease = function (details) {
+            var self = {};
+            assertProperties.assertRequired(details, ['id', 'app_id', 'active', 'app_name', 'profile_name',
+                'version']);
+            self.id = details.id;
+            self.active = ko.observable(details.active);
+            self.status = details.active ? gettext('Active') : gettext('Inactive');
+            self.app_name = details.app_name;
```

Comment:
Nitpick: why do the properties here mix casing styles? They should all be camel case.

Reply:
https://github.com/dimagi/commcare-hq/pull/24692/commits/5ae5494affcc192fea688a66d4ea716933ffae40